src/components/Atoms/Button/Button.js

```diff
@@ -4,6 +4,9 @@ import styled from 'styled-components';
 import spacing from '../../../theme/shared/spacing';
 import media from '../../../theme/shared/breakpoint';
 
+// Function to decide contrast color based on background color
+const getContrastColor = backgroundColor => (backgroundColor === 'red' ? 'white' : 'black');
```

Comment:
Rather than hardcoding these plain CSS colour values, let's continue to use the variables that you'll find in the [theme](https://github.com/comicrelief/component-library/blob/master/src/theme/crTheme/colors.js), much like how the now-commented ruled on line 32 does.